Responses/ChangeMembersResponse.cs

```diff
@@ -0,0 +1,11 @@
+﻿using PyrusApiClient;
+using System.Collections.Generic;
+
+namespace Pyrus.ApiClient.Responses
+{
+    public class ChangeMembersResponse : ResponseBase
+    {
+        public IEnumerable<Person> Members { get; set; } = new List<Person>();
```

Comment:
nit: stylistically stands out from other data contracts and there is not enough enter
Let's do it like this
```c#
public List<Person> Members { get; set; }

public List<string> Errors { get; set; }
```

Reply:
done